bin/jackal-test

```diff
@@ -2,4 +2,27 @@
 # -*- mode: ruby -*-
 # -*- encoding: utf-8 -*-
 
-require 'jackal/utils/spec/loader'
+require 'bogo-cli'
+require 'jackal'
+
+# This conditional is necessary since the cli parsing has a no-arg
+#   default of printing the help menu (not able to override the default)
+if ARGV.empty?
+  require 'jackal/utils/spec/loader'
+else
+  Bogo::Cli::Setup.define do
+    global_opts = lambda do
+      on :s, 'service_name=', 'Service to test (Eg: Jackal::Mail::Smtp - mail)'
```

Comment:
only issue with that is then you end up getting option params from (cli / slop) that look like `:"service-name"` which is a bit unexpected :neutral_face: